examples/piecestore-client/rpc/client/main.go

```diff
@@ -69,7 +69,7 @@ func main() {
 
 				id := client.NewPieceID()
 
-				if err := psClient.Put(context.Background(), id, dataSection, ttl); err != nil {
+				if err := psClient.Put(context.Background(), id, dataSection, ttl, "ABCD", "EFGH"); err != nil {
```

Comment:
You mean create an ID type?

Reply:
I don't know what an actual ID looks like yet so I have it just set to strings

Reply:
I meant to pick a more comprehensive hardcoded value. Like "payer-id" instead of "ABCD". It took a while until I figured out what "ABCD" and "EFGH" are.

Reply:
I see now. Updated